src/serac/infrastructure/accelerator.hpp

```diff
@@ -148,7 +148,7 @@ std::string getCUDAMemInfoString()
  * @tparam exec the memory space where the data lives
  * @param n how many entries to allocate in the array
  */
-template <typename T, ExecutionSpace exec>
+template <ExecutionSpace exec, typename T>
```

Comment:
Was there a reason that this was flipped? I know this is minor but scanning through most `ExecutionSpace exec` are the last template parameter.

Reply:
These arguments were rearranged so that it would be consistent with the variadic overload of the function defined immediately below this one. For C++ parsing reasons, variadic packs can only appear at the end of the explicitly-specified template argument list, so the ExecutionSpace has to come first.

Reply:
Excellent thanks for the explanation!